breadbox/breadbox/schemas/context.py

```diff
@@ -0,0 +1,15 @@
+from pydantic import Field, BaseModel
+
+
+class ContextSummary(BaseModel):
+    num_candidates: int
+    num_matches: int
+
+
+class Context(BaseModel):
+    # Context expression examples:
+    # - { "!": [ { "var": "slice/lineage/1/label" }, "Breast" ] }
+    # - { "==": [ { "var": "entity_label"}, "ACH-000001" ] }
+    # - { "==": [ {"var": "slice/growth_pattern/all/label"}, "Adherent" ] }
+    expr: dict[str, list]
```

Comment:
`expr` can also be a bool. That makes this special "All" option trivial to encode (`expr == True`).

![all](https://github.com/user-attachments/assets/3a05971b-73c5-4e44-a035-27fd3fca2dad)

[https://jsonlogic.com/#always-and-never](https://jsonlogic.com/#always-and-never)

Reply:
Oh, thank you! That's good to know. I'll update the typing now